load.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * SEO Module
+ *
+ * @package hm-platform/seo
+ */
+
+namespace HM\Platform;
+
+require_once __DIR__ . '/inc/namespace.php';
+
+add_action( 'hm-platform.modules.init', function () {
+	register_module( 'seo', __DIR__, 'SEO', [
```

Comment:
just a note on formatting, rather than breaking the function params onto multiple lines, I typically would pull the settings array into it's own definition

Reply:
I find it neater this way personally than declaring single-use variables but I don't feel that strongly about it. Happy to go for consistency across modules.